compiler/rustc_trait_selection/src/traits/project.rs

```diff
@@ -901,21 +939,21 @@ fn opt_normalize_projection_type<'a, 'b, 'tcx>(
             // evaluations can causes ICEs (e.g., #43132).
             debug!(?ty, "found normalized ty");
             obligations.extend(ty.obligations);
-            return Ok(Some(ty.value));
+            return Ok(Some(ty.value.into()));
```

Comment:
This goes hand in hand with the

```rust
Normalized {
                    value: normalized_ty.ty().unwrap(),
                    obligations: projected_obligations,
                }
```

further down. `value` should be `Term`, which should make a bunch of unwraps disappear.

Reply:
Oops I think I screwed myself when I rebased